oms-global/docker/docker-compose.yml

```diff
@@ -50,15 +50,18 @@ services:
     pgadmin:
         image: fenglc/pgadmin4:2.0
         restart: on-failure
-        expose:
-            - 5050
+        ports:
+            - 10100:5050
```

Comment:
let me crosscheck with what i have here because I am quite sure it is not needed to forward the port

Reply:
Forwarding the port allowed me to access it on appserver:1010 though, might be a nice additional feature to have (if for whatever reason you cant (or forget) to update your hosts file)